lib/login_page.dart

```diff
@@ -27,61 +23,198 @@ class LoginPageState extends ConsumerState<LoginPage> {
   }
 
   @override
-  Widget build(BuildContext context) {
+  Widget build(BuildContext context, WidgetRef ref) {
     return Scaffold(
-      body: Center(
+        body: Row(children: [
+      Container(
+        width: 800,
+        alignment: Alignment.center,
         child: Column(
           mainAxisSize: MainAxisSize.max,
           mainAxisAlignment: MainAxisAlignment.center,
-          children: <Widget>[
-            Text(
-              'Please Login',
-              textAlign: TextAlign.center,
-              overflow: TextOverflow.ellipsis,
-              style: const TextStyle(fontWeight: FontWeight.bold, fontSize: 24),
-            ),
+          children: [
+            Container(
+                margin: EdgeInsets.only(right: 270),
+                child: Text(
+                  'Log in',
+                  textAlign: TextAlign.left,
+                  overflow: TextOverflow.ellipsis,
+                  style: LoginStyle.titleStyle,
+                )),
             SizedBox(height: 50),
             ElevatedButton(
+                style: LoginStyle.buttonStyle,
                 onPressed: () {
                   signInWithGoogle().whenComplete(() {
                     //ref.read(isLogedIn.notifier).value = true;
                   });
                 },
-                child: const Text(
-                  "LogIn with Google",
-                  style: TextStyle(
-                      fontSize: 28.0,
-                      color: Color(0xFF0255ec),
-                      fontWeight: FontWeight.w500),
-                )),
+                child: Row(mainAxisSize: MainAxisSize.min, children: [
+                  Container(
+                    width: 50,
+                    height: 50,
+                    margin: EdgeInsets.only(right: 70),
+                    decoration: LoginStyle.containerStyle,
+                    child: Container(
+                        margin: EdgeInsets.only(right: 20),
+                        child:
+                            Image.asset("search.png", width: 30, height: 30)),
```

Comment:
not sure why this says search, not google.

also, can we avoid using assets and use Flutter icons for this? Or there is not Google icon available?

Reply:
There is no google icon available. I will change the name of the Image.